modules/ding_wayf_dk/ding_wayf_dk.module

```diff
@@ -0,0 +1,59 @@
+<?php
+/**
+ * @file
+ * This module is a support module to share function between it's sub-modules.
```

Comment:
No, this module file is used to make changes and share code across it's two sub-module in the modules folder (ding_wayf_dk_login and ding_wayf_dk_create_user).

It was an empty module file until support for gatewayf was requested by DDB.